src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReaderSettings.cs

```diff
@@ -3,25 +3,23 @@
 
 using System.IO;
 using System.Diagnostics;
-using System.Globalization;
-using System.Security;
+using System.Diagnostics.CodeAnalysis;
 using System.Xml.Schema;
 
 namespace System.Xml
 {
     // XmlReaderSettings class specifies basic features of an XmlReader.
     public sealed class XmlReaderSettings
     {
-        internal static readonly XmlReaderSettings s_defaultReaderSettings = new XmlReaderSettings() { ReadOnly = true };
+        internal static readonly XmlReaderSettings s_defaultReaderSettings = new() { ReadOnly = true };
 
         //
         // Fields
         //
 
         private bool _useAsync;
 
-
-        // Nametable
+        // NameTable
```

Comment:
you can remove these comments anyway unless they provide some useful information (`Fields` doesn't sound particularly useful)